compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -3311,3 +3324,22 @@ OMR::X86::CodeGenerator::considerTypeForGRA(TR::SymbolReference *symRef)
       return true;
       }
    }
+
+uint32_t
+OMR::X86::CodeGenerator::getOutOfLineCodeSize()
+   {
+   uint32_t totalSize = 0;
+
+   auto oiIterator = self()->getOutlinedInstructionsList().begin();
+   while (oiIterator != self()->getOutlinedInstructionsList().end())
+      {
+      uint32_t startOffset = (uint32_t)((*oiIterator)->getFirstInstruction()->getBinaryEncoding() - self()->getCodeStart());
+      uint32_t endOffset   = (uint32_t)((*oiIterator)->getAppendInstruction()->getBinaryEncoding() - self()->getCodeStart());
+
+      totalSize += endOffset - startOffset;
```

Comment:
Couldn't it just be:
```suggestion
      auto firstInstr = (*oiIterator)->getFirstInstruction()->getBinaryEncoding();
      auto nextInstr = (*oiIterator)->getAppendInstruction()->getBinaryEncoding();
      totalSize += (uint32_t)(nextInstr - firstInstr);
```
Since both offsets are determined by the same codestart? I believe you had done something similar above:
```
uint32_t blockSize = (uint32_t)(endCursor - startCursor);
```

Reply:
Yes! Copied from another place and forgot to modify. Fixed now. Also changed to static cast.